test/runnable/test15779.d

```diff
@@ -18,5 +18,6 @@ int main()
 
 void bar()
 {
+    version(CRuntime_Microsoft) {} else // TODO: fails for -O -inline, also disabled for the auto-tester
```

Comment:
What's the problem here?  What's the plan for eventually enabling it for the auto tester?  Is a bug report filed for it?

Reply:
Not sure, maybe related: https://issues.dlang.org/show_bug.cgi?id=13821 and https://issues.dlang.org/show_bug.cgi?id=16511.
I have added a new one: https://issues.dlang.org/show_bug.cgi?id=18547

Reply:
In that case I recommend adding links to those issues in the comments and come clean about you're understanding of what's going on.  Something like:

"Fibers appear to be broken in Windows.  We don't yet have a solution, see these issues...."  or something like that.  My motivations is to ensure that there is some kind of "paper trail" that one can follow to eventually solve the issue.

Reply:
I agree with the "paper trail" proposal, otherwise such comments just mystify the reader.

Reply:
@rainers my guess is that this test passes on LDC. Can you scope this to CRuntime_Microsoft and DigitalMars (i.e. dmd)?

Reply:
I think I figured it out: https://github.com/dlang/druntime/pull/2129. All windows builds should be affected, but happens spuriously.

Reply:
I have removed the workaround in the hope that the bugfix will be accepted.

Reply:
Thanks!